docs/operator-docs/update-cumulus-dependencies.md

```diff
@@ -62,3 +62,20 @@ Note that there may be configuration changes required with a new version. Please
 When breaking changes have been introduced, the Cumulus Core team will publish a document with instructions on migrating from one version to another. *IMPORTANT:* When upgrading through many versions, each migration should be done in the order of release (if going from version 1.z.0 from 1.x.0, 1.x.0 -> 1.y.0 -> 1.z.0).
 
 The migration documents published on release can be found [here](https://nasa.github.io/cumulus/docs/upgrade/upgrade-readme).
+
+## Maintaining the current Version of a Dependency
+
+If you wish to maintain the current version for a specific dependency, make sure to save that version to the package.json with this command.
+
+```shell
+npm install [<@scope>/]<name>@<version> --save --save-exact
+```
+
+Or update that dependency version directly in your package.json to include no prefix.
```

Comment:
I don't think we should recommend editing `package.json` directly. If you do that, then the lockfile is only updated when you later run `npm install` for the project instead of when you `npm install package@1.2.3 --save`, which makes the context of the changes to the lockfile a bit more confusing